src/containers/App.js

```diff
@@ -158,12 +158,15 @@ export default class App extends React.Component {
         path={path}
         search={doSearch}
         opened={leftNavIsOpen}
-        networkNavAnimation={moveWithMenu}
-        networkNavEasing={leftNavEasing}
         />}
 
       <VelocityComponent animation={moveWithMenu} easing={leftNavEasing}>
         <div id='main'>
+          {currentUser && !isMobile && networkCommunities &&
```

Comment:
this is a lot of clauses; can we just wrap them up into a `showNetworkNav` const?